decidim-stats/config/locales/ca.yml

```diff
@@ -0,0 +1,10 @@
+ca:
+  decidim:
+    stats:
+      authorization_exports:
```

Comment:
@mllocs translations are pending? All copied from English

Reply:
oh, right. I can do the translations.

Reply:
Done